test/helpers.go

```diff
@@ -1,80 +1,70 @@
 package test
 
 import (
+	"bytes"
+	"errors"
 	"fmt"
-	"io/ioutil"
-	"os"
 	"os/exec"
-	"path/filepath"
 	"regexp"
 	"testing"
+
+	"github.com/cli/cli/internal/run"
 )
 
 // OutputStub implements a simple utils.Runnable
 type OutputStub struct {
-	Out []byte
+	Out   []byte
+	Error *run.CmdError
```

Comment:
Should this be of type `error` so we can return any error type that we want, instead of just `*run.CmdError`?

Reply:
like i said above I misread the code as always using `CmdError` and didn't see the need for that flexibility; i'll update it.